test-plone-4.1.x.cfg

```diff
@@ -3,3 +3,6 @@ extends =
     https://raw.github.com/4teamwork/ftw-buildouts/master/test-plone-4.1.x.cfg
 
 package-name = ftw.shop
+
+[versions]
+path.py = 8.1.2
```

Comment:
Put a version requirement in `setup.py`. Something like this: https://github.com/4teamwork/ftw.shop/blob/master/setup.py#L53
The actual version pinnings are made by the policies to ensure a consistent state of the project.